federation/pkg/federation-controller/util/meta_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"testing"
+
+	api_v1 "k8s.io/kubernetes/pkg/api/v1"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestObjectMeta(t *testing.T) {
+	o1 := api_v1.ObjectMeta{
```

Comment:
@mwielgus Please use table driven testing as per Kubernetes conventions:

https://github.com/kubernetes/kubernetes/blob/master/docs/devel/coding-conventions.md#testing-conventions

and as per example that I sent in previous PR comment:

https://github.com/kubernetes/kubernetes/blob/master/federation/pkg/dnsprovider/dns_test.go#L53